extension/test/newman/collection.json

```diff
@@ -129,6 +215,93 @@
           },
           "response": []
         },
+        {
+          "name": "Delete: Customer favorite",
+          "event": [
+            {
+              "listen": "test",
+              "script": {
+                "id": "29b3580c-acf6-4360-abd4-6c3edd110e8d",
+                "type": "text/javascript",
+                "exec": [
+                  "pm.test(\"Status code is 200\", function () {",
+                  "    pm.response.to.have.status(200);",
+                  "});",
+                  "",
+                  "pm.environment.set(\"wishlist-item-id\", response.products[0].id);"
+                ]
+              }
+            }
+          ],
+          "request": {
+            "method": "POST",
+            "header": [
+              {
+                "key": "Content-Type",
+                "value": "application/json"
+              }
+            ],
+            "body": {
+              "mode": "raw",
+              "raw": "{\n\t\"productId\": \"{{wishlist-item-id}}\"\n}"
+            },
+            "url": {
+              "raw": "{{domain}}{{endpoint_user_favorites_delete}}",
+              "host": [
+                "{{domain}}{{endpoint_user_favorites_delete}}"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "Get: Customer favorites Copy",
+          "event": [
+            {
+              "listen": "test",
+              "script": {
+                "id": "535f151b-3bb3-4b77-80ef-f77b56c068fd",
+                "type": "text/javascript",
+                "exec": [
+                  "pm.test(\"Status code is 200\", function () {",
+                  "    pm.response.to.have.status(200);",
+                  "});",
+                  "",
+                  "const response = pm.response.json();",
+                  "pm.test(\"Response have the correct format\", () => {",
+                  "    pm.expect(response.products).to.be.a('array').to.not.be.empty;",
+                  "});",
+                  "",
+                  "pm.test(\"One wishlist item was removed\", () => {",
+                  "    pm.expect(response.products.length).to.eql(pm.environment.get(\"wishlist-length\")-1);",
+                  "});",
+                  "",
+                  "pm.environment.set(\"wishlist-length\", response.products.length);"
```

Comment:
not sure why u need this, you also need to clean up your created variables at the end